apps/base/utils.py

```diff
@@ -69,3 +75,30 @@ def decode_data(data):
     for i in data:
         decoded.append(base64.decodestring(i+"=="))
     return decoded
+
+
+def send_data(webhook, message):
+    """
+    Posts message to url
+    """
+    try:
+        requests.post(
+            webhook,
+            data=json.dumps({'text': str(message)}),
+            headers={'Content-Type': 'application/json'}
+        )
+    except Exception as e:
+        logger.info(
+            'Exception raised while sending slack notification. \n Exception message: {}'.format(e))
+
+
+def send_slack_notification(webhook=settings.SLACK_WEBHOOKS['default'], message=""):
```

Comment:
@yashdusing 
`send_slack_notification` should only run if the Project is not running in `Dev`. Add a condition for it